runtime/proto/service/log_replication.proto

```diff
@@ -52,4 +52,23 @@ enum LogReplicationEntryType {
   SNAPSHOT_REPLICATED = 4;
   SNAPSHOT_END = 5;
   SNAPSHOT_TRANSFER_COMPLETE = 6;
+}
+
+message LogReplicationSession {
+  string sinkClusterId = 1;
```

Comment:
.proto files typically use the _ convention so it can translate getters and setters specific to the language convention -  if you change this to sink_cluster_id the Java code should not require any changes